Automattic/wp-calypso
# Add Cloudflare cards to Home, Stats, and Marketing/Traffic

This PR implements @sfougnier's final Cloudflare card layouts details in pbAPfg-1fn-p2.

The changes include:
* New task illustration for the card on My Home.
* Upgrade nudges added to the Cloudflare cards on the Settings and Marketing pages.
* Final destination URLs for the learn more links.

Screenshots:

The Cloudflare task card on My Home:
<img width="968" alt="Screen Shot 2021-02-16 at 12 54 00 PM" src="https://user-images.githubusercontent.com/35781181/108104454-524b5100-7059-11eb-9abb-b53f34dcd1f0.png">

Learn more banner on Stats:
<img width="968" alt="Screen Shot 2021-02-16 at 12 58 51 PM" src="https://user-images.githubusercontent.com/35781181/108104634-8fafde80-7059-11eb-8a81-47e5f489760e.png">

Upgrade banner on Stats:
<img width="968" alt="Screen Shot 2021-02-16 at 12 59 03 PM" src="https://user-images.githubusercontent.com/35781181/108104882-df8ea580-7059-11eb-9bc4-ec7b0464cefd.png">

Marketing > Traffic page:
<img width="968" alt="Screen Shot 2021-02-16 at 12 59 45 PM" src="https://user-images.githubusercontent.com/35781181/108105135-3ac09800-705a-11eb-881c-d08fc6c8f7f3.png">

Settings > Performance (upsell view):
<img width="968" alt="Screen Shot 2021-02-16 at 1 15 52 PM" src="https://user-images.githubusercontent.com/35781181/108105836-2204b200-705b-11eb-8301-c5a50c882625.png">

Settings > Performance (Learn more view):
<img width="1071" alt="Screen Shot 2021-02-16 at 1 23 31 PM" src="https://user-images.githubusercontent.com/35781181/108105918-406aad80-705b-11eb-8f6f-d71ac20fe5a2.png">


#### Testing instructions

* Patch your sandbox with 55161-code.
* Sandbox public-api.
* Checkout this PR.
* Force Cloudflare task card to show on My Home via `/home/SITESLUG?dev=true&view=VIEW_CLOUDFLARE`
* Navigate to `/stats/day/SITESLUG` and with Premium and Free plan accts and confirm that the appropriate banner shows (upsell vs learn more)
